{"package_name": "java.time", "subclass_list": ["DateTimeParseException", "UnsupportedTemporalTypeException", "ZoneRulesException", "DateTimeParseException", "UnsupportedTemporalTypeException", "ZoneRulesException"], "class_description": "Exception used to indicate a problem while calculating a date-time. This exception is used to indicate problems with creating, querying\n and manipulating date-time objects. ", "class_type": "class", "typealias": [], "interface_list": [""], "class_inherit_list": ["RuntimeException"], "Methods": [{"params": [{"param_name": ["message"], "param_type": "String", "param_description": " the message to use for this exception, may be null"}], "class_name": "DateTimeException", "is_static": false, "return_value": {"return_name": "", "return_description": [], "return_type": []}, "method_name": "init", "method_description": "Constructs a new date-time exception with the specified message.\n", "method_declar": ""}, {"params": [{"param_name": ["message"], "param_type": "String", "param_description": " the message to use for this exception, may be null"}, {"param_name": ["cause"], "param_type": "Throwable", "param_description": " the cause of the exception, may be null"}], "class_name": "DateTimeException", "is_static": false, "return_value": {"return_name": "", "return_description": [], "return_type": []}, "method_name": "init", "method_description": "Constructs a new date-time exception with the specified message and cause.\n", "method_declar": ""}], "class_name": "DateTimeException", "Vars": null}